CODEOWNERS

```diff
@@ -1 +1 @@
-.github/workflows/jira.yaml @hashicorp/vault-ecosystem
+/ @hashicorp/vault-ecosystem
```

Comment:
You may have already received an alert for this, but looks like we should remove the slash.

Reply:
I think it should be `*` in retrospect.